### ISSUE TITLE
add new facebook RTMPS functionality

### ISSUE THREAD

**Issue by fikri2992** [JasonRivers/Docker-nginx-rtmp#19]
since facebook deprecating rtmp to rtmps can this repo support rtmps ?

**Comment by JasonRivers** [comment]
This is something I have not yet tried to do, but it's something that I have thought about. I believe you should be able to create an RTMPS push rule, but I have not yet tried,

**Comment by fikri2992** [comment]
this would be helpful if you can update this with some RTMPS handle, I have read that Nginx RTMP still not support using RTMPS but the community seems using Stunnel which I try worked but don't know how to integrate to docker 


here tutorial I have tried and it seems to works for me, but idk how to integrate with docker
https://dev.to/lax/rtmps-relay-with-stunnel-12d3

**Comment by JasonRivers** [comment]
Thanks for that link, That's very helpful!

**Comment by fikri2992** [comment]
no, actually I'm the one who needs to say thanks this repo saves me a lot of time 😅😅
looking forward to seeing an update for facebook RTMPS

**Comment by kkrishnan90** [comment]
Any update on Facebook RTMPS? Eagerly waiting to look at it as currently there's a necessity for me to include it in my project. Everything else this docker is amazingly beautiful!

**Comment by jessicah** [comment]
I've created a docker-compose example for supporting RTMPS: https://github.com/jessicah/nginx-rtmp-stunnel

Can probably use the env variables this container uses rather than specifying an nginx.conf file explicitly, but it was easier for testing.

A more polished version would use a similar shell script to configure the stunnel container the same way nginx is done (e.g. multiple RTMPS endpoints to push to).
